AdobeStockAsset/Test/Api/AssetRepository/DeleteTest.php

```diff
@@ -67,16 +67,16 @@ public function testDelete(): void
      */
     public function testDeleteWithException(): void
     {
+        $notExistedAssetId = -1;
         try {
-            $notExistedAssetId = -1;
             $this->deleteAsset($notExistedAssetId);
             $this->fail('Expected throwing exception');
         } catch (\Exception $e) {
-            if (TESTS_WEB_API_ADAPTER === self::ADAPTER_REST) {
+            if (constant('TESTS_WEB_API_ADAPTER') === self::ADAPTER_REST) {
```

Comment:
I don't think that constant('TESTS_WEB_API_ADAPTER') is better than TESTS_WEB_API_ADAPTER.

Is there any way to point phpstan to the constant declaration?

Reply:
There is no TESTS_WEB_API_ADAPTER declaration in PHP.
So, in any case, PHPStan wouldn't find it. PHPStorm validates it as undefined as well. Actually wrap it in `constant()` is a good idea because if it would be actually undefined `constant()` call will return `null` while raw `TESTS_WEB_API_ADAPTER` will throw php warning or error.